src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -30,21 +60,7 @@
                   "startLine": 10
                 }
               },
-              "logicalLocation": [
-                {
-                  "name": "app",
-                  "kind": "android-module"
-                },
-                {
-                  "name": "microsoft.androidstudioformatexample",
-                  "kind": "package"
-                },
-                {
-                  "name": "microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()",
-                  "kind": "method"
-                }
-              ],
-              "fullyQualifiedLogicalName": "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()"
+              "logicalLocation": "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()"
```

Comment:
right, forgot this. sure, i'm ok with logicalLocation


---
In reply to: [60291866](https://github.com/Microsoft/sarif-sdk/pull/196#discussion_r60291866) [](ancestors = 60291866)